hybridse/include/node/sql_node.h

```diff
@@ -1819,12 +1815,8 @@ class CreateStmt : public SqlNode {
 
     bool GetOpIfNotExist() const { return op_if_not_exist_; }
 
-    int GetReplicaNum() const { return replica_num_; }
-
-    int GetPartitionNum() const { return partition_num_; }
-
-    NodePointVector &GetDistributionList() { return distribution_list_; }
-    const NodePointVector &GetDistributionList() const { return distribution_list_; }
+    NodePointVector &GetTableOptionList() { return table_option_list_; }
```

Comment:
const function

Reply:
have overloaded an const function of `GetTableOptionList` in line 1819

Reply:
misleading naming. if u want to change `table_option_list_` inside class
1. should return pointer instead
2. add a `mutable` prefix, e.g `NodePointVector* MutableTableOptionList`

Reply:
if it is a overloaded function, add `override` label

Reply:
not a overloaded function, `GetTableOptionList` will const and not change `tablet_option_list_`, another function is `MutableTableOptionList` will change `tablet_option_list_`